src/shell/Shell_test.cpp

```diff
@@ -87,6 +87,15 @@ TEST_CASE("shell.builtin.get_variable")
     shell("$BRU");
 }
 
+TEST_CASE("shell.builtin.get_variable_inside_curl_brackets")
+{
+    TestShell shell;
+    shell("set BRU hello");
+    CHECK(shell.env.get("BRU").value_or("NONE") == "hello");
+    fmt::print("=======================================\n");
+    shell("${BRU}");
```

Comment:
I think here's missing something. this line (alone) doesn't make sense.

Reply:
I used it to test in debug output that IR is generated properly, at the moment we can not test generated IR unfortunately